### ISSUE TITLE
with_sequence skips if count=1

### ISSUE THREAD

**Issue by memelet** [ansible/ansible#11422]
ansible 1.9.2
osx and ubuntu

```
    - name: count=2
      debug: var=item
      with_sequence: count=2

    - name: count=1
      debug: var=item
      with_sequence: count=1
```

results in:

```
TASK: [count=2] ***************************************************************
ok: [localhost] => (item=1) => {
    "item": "1",
    "var": {
        "item": "1"
    }
}
ok: [localhost] => (item=2) => {
    "item": "2",
    "var": {
        "item": "2"
    }
}

TASK: [count=1] ***************************************************************
skipping: [localhost]
```

**Comment by memelet** [comment]
This check at the end of run is what is preventing count=1 from executing. But its not clear to me what some of the other changes made which adjust start,end,stride are trying accomplish. Are there unit-tests for plugins? I can't seem to find any.

```
                results.extend(self.generate_sequence())
```

to 

```
                if self.start != self.end:
                    results.extend(self.generate_sequence())
```

**Comment by memelet** [comment]
For version 2816b8679ba652c1631a21e0361010bc9e817aaa count=1 works. So this regression was introduced in the last commit to this file: 54fb04afc42a43d5951742bf86eccc01be104437

**Comment by bcoca** [comment]
yes, that was my fix for another issue with sequence, i added several tests to the repo but never tried count=1, I'll push a fix (and a new test) soon.

**Comment by bcoca** [comment]
^fix should make 1.9.3, porting to current devel also

**Comment by bcoca** [comment]
closing via 5122455db833eeddc92b74c44d112c125878502b
